presto-docs/src/main/sphinx/release/release-302.rst

```diff
@@ -0,0 +1,63 @@
+===========
+Release 302
+===========
+
+General Changes
+---------------
+
+* Fix cluster starvation when wait for minimum number of workers is enabled. (:issue:`155`)
+* Fix backup of queries blocked waiting for minimum number of workers. (:issue:`155`)
+* Fix failure when preparing statements that contain a quoted reserved word as a table name. (:issue:`80`)
+* Fix query failure when spilling is triggered during certain phases of query execution. (:issue:`164`)
+* Add :doc:`/connector/elasticsearch`. (:issue:`118`)
+* Add support for ``boolean`` type to :func:`approx_distinct`. (:issue:`82`)
+* Add support for boolean columns to ``EXPLAIN`` with type ``IO``. (:issue:`157`)
+* Add SphericalGeography type and related functions. (:issue:`166`)
+* Remove deprecated system pool. (:issue:`168`)
+* Improve query performance for certain queries involving ``ROLLUP``. (:issue:`105`)
+* Preserve table name quoting in the output of ``SHOW CREATE VIEW``. (:issue:`80`)
+
+CLI Changes
+-----------
+
+* Add ``--trace-token`` option to set the trace token. (:issue:`117`)
+* Add spilled data size debug output. (:issue:`161`)
+
+Web UI Changes
+--------------
+
+* Add spilled data size to query details page. (:issue:`161`)
+
+Security Changes
+----------------
+
+* Add ``http.server.authentication.krb5.principal-hostname`` configuration option to set the hostname
+  for the Kerberos service principal. (:issue:`146`, :issue:`153`)
+* Add support for client-provided extra credentials that can be utilized by connectors. (:issue:`124`)
+
+Hive Connector Changes
+----------------------
+
+* Fix Parquet predicate pushdown for ``smallint``, ``tinyint`` types. (:issue:`131`)
+* Add support for Google Cloud Storage (GCS). Credentials can be provided globally using the
+  ``hive.gcs.json-key-file-path`` configuration property, or as a client-provided extra credential
+  named ``hive.gcs`` if the ``hive.gcs.use-access-token`` is enabled. (:issue:`124`)
+* Fix support for tables with `external_location` property pointing to an empty S3 directory. (:issue:`75`)
```

Comment:
Double backticks

Reply:
Maybe change to
```
* Allow creating tables with the ``external_location`` property pointing to an empty S3 directory. (:issue:`75`)

Reply:
We weren't sure whether the fix was about creating tables that look like that or running queries over such tables.

Reply:
It is for creating tables

> If you try to create a table with this S3 prefix you will get an
"External location must be a directory" error.

I also read the code

Reply:
The problem is we verify the location when creating the table and that check was failing due to not being a directory that exists.